doc/api/cli.md

```diff
@@ -3181,6 +3181,10 @@ added: v22.1.0
 Enable the [module compile cache][] for the Node.js instance. See the documentation of
 [module compile cache][] for details.
 
+### `NODE_COMPILE_CACHE_RELATIVE_PATH=0`
```

Comment:
I wonder if "relative" is more of an implementation detail, what we really want to allow is "portable" compile cache (i.e. you can move the code and the compile cache to a different base directory, the cache will still be hit as long as they maintain the same directory structure) So maybe `NODE_COMPILE_CACHE_PORTABLE=0` would be more intuitive and we can explain about the directory structure with an example in the docs.

Reply:
I concur.

Also, why is this not the default?

Reply:
I think it would be nice to eventually make it the default, though it would be more cautious to make it opt-in first - there are some points that might need some battle testing, for example, we are keying by filenames but in practice not all "filenames" are filenames (source maps, synthetic IDs, and of course...monkey patchers), and we need to see whether this just leads to more cache misses or can lead to a stability issue.